aspnetcore/client-side/spa/react-with-redux.md

```diff
@@ -13,10 +13,12 @@ uid: spa/react-with-redux
 ::: moniker range="= aspnetcore-2.0"
 
 > [!NOTE]
-> This documentation isn't about the React-with-Redux project template included in ASP.NET Core 2.0. It's about the newer React-with-Redux template to which you can update manually. The template is included in ASP.NET Core 2.1 by default.
+> This documentation isn't about the React-with-Redux project template included in ASP.NET Core 2.0. It's about the newer React-with-Redux template to which you can update manually. The template is included in ASP.NET Core 2.1 and later.
```

Comment:
This predates the PR, but this lingo seems a bit too loose for APEX. :smile: lol

Also, "to which" phrasing is odd.

```suggestion
> This documentation doesn't pertain to the React-with-Redux project template included in ASP.NET Core 2.0. It pertains to the newer React-with-Redux template that you can update manually. The template is available in ASP.NET Core 2.1 or later.
```